src/platform/packages/shared/kbn-ui-actions-browser/src/service/ui_actions_service.ts

```diff
@@ -40,8 +42,16 @@ export class UiActionsService {
     this.triggers = triggers;
     this.actions = actions;
     this.triggerToActions = triggerToActions;
+    this.executionService = new UiActionsExecutionService();
   }
 
+  protected readonly getNotificationsService = () => this.notificationsService;
+
+  public readonly start = (core: CoreStart) => {
```

Comment:
The `start` method is called by the plugin once the core start lifecycle is available.